cart.html

```diff
@@ -88,246 +90,17 @@
-				}
-				else {
-					$(".jQuerySimpleCart").append($(".jQuerySimpleCartDetails .cartTotals").clone());
-				}
-				$(".jQuerySimpleCart .jQuerySimpleCart-empty").hide();
-			}
-			else {
-				$(".jQuerySimpleCart .jQuerySimpleCart-empty").show();
-				$(".jQuerySimpleCart .cartTotals").hide();
-			}
-			
-			//add in-cart class
-			$(productBlockSelector).each(function() {
-				var $productBlock = $(this);
-				var isInCart = false;
-				$(products).each(function(k, product) {
-					if($productBlock.find(productTitleSelector).text() == product.title) {
-						isInCart = true;
-					}
-				});
-				if(isInCart) {
-					$productBlock.addClass("jQuerySimpleCart-inCart");
-				}
-				else {
-					$productBlock.removeClass("jQuerySimpleCart-inCart");
-				}
-			});
-		}
-		
-		function getCartProducts() {
-			var products = [];
-			$(".jQuerySimpleCartDetails ul.products li").each(function(k, el) {
-				var product = {
-					'title'	: $(el).find(".title").text(), 
-					'price'	: $(el).find(".price").text().replace(/[^0-9,.]/g, "").replace(",", "."),
-					'qty'	: parseInt($(el).find(".qty input").val())
-				}
-				if(product.qty > 0) {
-					products.push(product);
-				}
-			});
-			return products;
-		}
-		
-		function saveCartFromLayout() {
-			var products = getCartProducts();
-			setCookie("jQuerySimpleCart", JSON.stringify(products), cartCookieDays);
-		}
-		
-		function addToCart(product) {
-			var products = getCookie("jQuerySimpleCart") && JSON.parse(getCookie("jQuerySimpleCart")) || [];
-			if(products.length) {
-				isExistingProduct = false;
-				$(products).each(function(k, aProduct) {
-					if(product.title == aProduct.title) {
-						aProduct.qty = aProduct.qty + product.qty;
-						isExistingProduct = true;
-					}
-				});
-				if(!isExistingProduct) {
-					products.push(product);
-				}
-			}
-			else {
-				products.push(product);
-			}
-			console.log("Product added: ", product);
-			setCookie("jQuerySimpleCart", JSON.stringify(products), cartCookieDays);
-			buildCartLayout();
-		}
-		
-		function clearCart() {
-			document.cookie = 'jQuerySimpleCart=; Path=/; Expires=Thu, 01 Jan 1970 00:00:01 GMT;';
-			buildCartLayout();
-		}
-		
-		function setCookie(name, value, days) {
-			var expires = "";
-			if (days) {
-				var date = new Date();
-				date.setTime(date.getTime() + (days*24*60*60*1000));
-				expires = "; expires=" + date.toUTCString();
-			}
-			document.cookie = name + "=" + (value || "")  + expires + "; path=/";
-		}
-		
-		function getCookie(name) {
-			var nameEQ = name + "=";
-			var ca = document.cookie.split(';');
-			for(var i=0;i < ca.length;i++) {
-				var c = ca[i];
-				while (c.charAt(0)==' ') c = c.substring(1,c.length);
-				if (c.indexOf(nameEQ) == 0) return c.substring(nameEQ.length,c.length);
-			}
-			return null;
-		}
-	});			
-</script>
\ No newline at end of file
+<script type="text/javascript" src="cart.js"></script>
```

Comment:
Please rename all the files in some specific, like `simple-cart` etc..

Reply:
ok